items.json

```diff
@@ -164080,14 +164080,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_steppe_spiked_mace_v1_h0",
-    "baseId": "crpg_steppe_spiked_mace_v1",
+    "id": "crpg_steppe_spiked_mace_v2_h0",
```

Comment:
speed reduction is OK but should be offset by an increase in damage to match tiers